package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java

```diff
@@ -252,6 +252,18 @@ enum Option {
     VRANG_CLOUD_PROXY_NAME(
             "varng_cloud_proxy_name",
             ConfigurationVraNg.CLOUD_PROXY_NAME),
+
+	/**
+	 * VRANG unrelease blueprint versions. Decides wether old versions need to be
+	 * unrelased
+	 *
+	 * This only works when running in non interactive mode
+	 */
+	VRANG_BP_UNRELEASE_VERSIONS(
+			"vrang_bp_unrelease_versions",
+			ConfigurationVraNg.UNRELEASE_BLUEPRINT_VERSIONS),
+
+
```

Comment:
Are you sure that this is a part of this PR, I think you need to revert this, and make it a part of another PR in order not to pollute the logic of this one.

Reply:
It's really really small... and I didn't want to go and create a special Pr for it. Can we make an exception?